operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -21,6 +21,7 @@ description: 50% Draft
   * Marketing reviews
```

Comment:
I heard from AmyN that Ian is hoping we change this to match our T-XX process dates. If she hasn't reached out to you, can you help ask for more details?

E.g.  
 - T-XX: Writer drafts announce with feedback
 - T-XX: Release manager checks for spelling mistakes, ... 
 - ...